keyboards/e7v1/info.json

```
@@ -0,0 +1,26 @@
+{
+    "keyboard_name": "E7 - V1",
```

Looks funny with too many white spaces

```suggestion
    "keyboard_name": "E7-V1",
```